cg/server/api.py

```diff
@@ -520,6 +520,21 @@ def get_order(order_id: int):
         return make_response(jsonify(error=str(error)), HTTPStatus.NOT_FOUND)
 
 
+@BLUEPRINT.route("/orders/<order_id>/delivery_message")
+def get_delivery_message_for_order(order_id: int):
+    """Return the delivery message for an order."""
+    try:
+        response: DeliveryMessageResponse = delivery_message_service.get_delivery_message_for_order(
+            order_id
+        )
+        response_dict: dict = response.model_dump()
+        return make_response(response_dict)
+    except IndexError as error:
```

Comment:
This `IndexError` seems a little mystical to me, if you think it makes sense maybe a more specific exception can be raised

Reply:
Yeah, I will wrap it in something more descriptive!